### PR TITLE
config.py: Divert help command to private message.

### PR DESCRIPTION
**Pull request by meetmangukiya**
Adds to https://github.com/coala/corobo/issues/32

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -20,3 +20,5 @@
 BOT_IDENTITY = {
     'token': os.environ['COBOT_TOKEN']
 }
+
+DIVERT_TO_PRIVATE = ('help', )
```
Comment (Udayan12167) [inline config.py]: So this means if I type `corobo help` I'll receive a PM of the help command. Is it possible to point this out to the user with one sentence in that group itself?

Comment (meetmangukiya) [inline config.py]: yes.
Not sure if it is possible, this is built-in command

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/135/commits/6edeb7468cafd014203141dcc45b7a9da925f1b8

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**